libs/lwt_wrapper/Lwt_wrapper.mli

```diff
@@ -0,0 +1 @@
+val run : 'a Lwt.t -> 'a
```

Comment:
Please comment what is doing this function, and how it differs from Lwt_main.run and why
we need this intermediate thing,